workspaces/host/Cargo.toml

```diff
@@ -42,5 +42,5 @@ lto = true
 opt-level = 3
 
 [patch.crates-io]
```

Comment:
Should we add `ring` patch?

Reply:
It's not strictly necessary. The changes we have for `ring` are only needed inside an enclave. `linux-host` is outside.
It might be good just to keep things consistent.

Reply:
You make a good point. We can leave it like this and change if we encounter an issue.